src/app/containers/ChartbeatAnalytics/index.test.jsx

```diff
@@ -9,10 +9,14 @@ import * as testUtils from '../../lib/analyticsUtils/chartbeat';
 import * as utils from '../../lib/analyticsUtils';
 import * as amp from './amp';
 import * as canonical from './canonical';
-import { localBaseUrl } from '../../../testHelpers/config';
 import frontPageData from '../../../../data/news/frontpage';
 
 const defaultToggleState = {
+  local: {
+    chartbeatAnalytics: {
+      enabled: false,
```

Comment:
Your adding this as false, but then theres no `local` test? Can you duplicate the test one, for local please

Reply:
test case added now.